pkg/auth0-cli-config-generator/main.go

```diff
@@ -1,224 +0,0 @@
-// auth0-cli-config-generator: A command that generates a valid config file that can be used with auth0-cli.
```

Comment:
it feels so good
![](https://media.giphy.com/media/glvyCVWYJ21fq/giphy-downsized.gif)